avogadro/qtplugins/plugindownloader/downloaderwidget.h

```diff
@@ -0,0 +1,83 @@
+#ifndef AVOGADRO_DOWNLOADERWIDGET_H
+#define AVOGADRO_DOWNLOADERWIDGET_H
+
+#include <QtNetwork/QNetworkAccessManager>
```

Comment:
It is good practice to minimize what is included in the header, predeclaring where reasonable, this doesn't matter as much in plugins as they are not reused but it is worth bearing in mind.